package.json

```diff
@@ -201,7 +201,7 @@
     "yosay": "^1.2.1"
   },
   "devDependencies": {
-    "@ciscospark/eslint-config": "^1.0.0",
+    "@ciscospark/eslint-config": "^2.0.0",
```

Comment:
wait, really? nothing was supposed to be published from the new repo until this PR merged.